src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_RANDOM_LENGTH = 100;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int AMOUNT_OF_FIGURES = 5;
+    private final Random random = new Random();
+
+    public Figure getRandomFigure() {
+        String color = new ColorSupplier().getRandomColor();
```

Comment:
Introduce `ColorSupplier` instance as field variable, you don't need to create new object every time you call `getRandomFigure`